doc/architecture/sensor_view_configuration.adoc

```diff
@@ -0,0 +1,15 @@
+= Sensor view configuration
+
+The sensor view configuration configures the sensor view.
+There are two sources for sensor-view-configuration data.
+
+Sensor-view-configuration data can be provided by the environment simulation.
+In this case, the data is intended for the automatic configuration of an environment simulation in order to supply the necessary input to a sensor model.
+Sensor-view-configuration data is not intended to provide parameters to a generic sensor model. 
```

Comment:
```suggestion
Sensor-view-configuration data is not intended to provide exact parameters to a sensor model. 
```

Reply:
CCB Meeting 04.08.2021:
- Sensor-view-configuration data is not intended to provide parameters to a sensor model.